src/rules/typescript.ts

```diff
@@ -8,6 +8,8 @@ import memberOrderingList from './rule-helpers/member-ordering-list';
 export default {
   // Enforces function overloads to be consecutive.
   'adjacent-overload-signatures': true,
+  // Bans specific types from being used. Does not ban the corresponding runtime objects from being used.
+  'ban-types': false,
```

Comment:
Hahaha 😆 
Will this rule even flag that? `$` wouldn't be passed around as a type, would it?